components/faqs/nesting.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withTheme, ThemeProvider } from 'styled-components'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+import LiveEdit from '../LiveEdit'
+
+const scope = { withTheme, ThemeProvider }
+const sample = (`
+const ColorChanger = styled.section\`
+  background: papayawhip;
+
+  > h2 {
+    color: palevioletred;
+  }
+
+  @media(min-width: 768px) {
+    background: mediumseagreen;
+
+    > h2 {
+      color: papayawhip;
+    }
+  }
+\`;
+render(
+  <ColorChanger href="#">
+    <h2>Hello world!</h2>
+  </ColorChanger>
+)
+`).trim()
+
+const Nesting = () => (
+  <SectionLayout title="Nesting">
```

Comment:
The problem is, that a "Best Practices" section that I'm thinking of shouldn't be too restrictive, but just include some general rules and recommendations. Otherwise people will think that they can't use it, when they don't agree with some of them.

I wouldn't call it "Tips & Tricks". I've absorbed a lot of them into the other sections and new sub sections. Oftentimes these are so important that they're basics.

The FAQ section is for any question that is not sufficiently addressed by the rest of the docs, and are thus "outstanding questions", in this sense. This is why I'd rephrase it, because the problem is that people would ask these questions anyway on GitHub, Gitter, or Twitter, if we don't answer them.

If we make this section into anything else than an FAQ, we'll lose control over our structure, because it's very tempting to treat such a section as a "Miscellaneous" section, to put everything into it, that takes too long to add somewhere else.

If you think about it, the following sections could also be regarded as "Tips & Tricks", but are integrated into the "narrative" of the Basics and Advanced intros/sections:

- Animations
- Refs
- Security
- Media Templates
- Flow & TypeScript

Even though these are now neatly inside the other sections, they used to be separate. ;)